.travis.yml

```diff
@@ -100,16 +100,6 @@ jobs:
       env:
         - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
       if: type IN (push, pull_request)
-    - stage: compile
-      script: make libs
-      addons:
-        apt:
-          sources:
```

Comment:
just remove this line and below line. Do same for the test stage.

Error comes because default changed from trusty to xenial: https://travis-ci.community/t/cannot-apt-get-install-clang-5-0/3250

Reply:
This can still be fixed, so that we do compile for clang5 except with that removal of these two lines here